src/components/NavBar/MenuItemMobile/index.tsx

```diff
@@ -0,0 +1,184 @@
+import React, { useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import {
+  Button,
+  createStyles,
+  Grid,
+  Switch,
+  Theme,
+  Typography,
+  withStyles,
+  WithStyles,
+} from '@material-ui/core';
+import Accordion from '@material-ui/core/Accordion';
+import AccordionSummary from '@material-ui/core/AccordionSummary';
+import AccordionDetails from '@material-ui/core/AccordionDetails';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCaretDown } from '@fortawesome/free-solid-svg-icons';
+
+import { LayerType, TableType } from '../../../config/types';
+import { addLayer, removeLayer } from '../../../context/mapStateSlice';
+import { loadTable } from '../../../context/tableStateSlice';
+import { layersSelector } from '../../../context/mapStateSlice/selectors';
+import { menuList } from '../utils';
+
+function MenuItemMobile({ classes }: MenuItemMobileProps) {
+  const [expanded, setExpanded] = useState<string>('');
+
+  const selectedLayers = useSelector(layersSelector);
+
+  const dispatch = useDispatch();
+
+  const handleChange = (panel: string) => (
+    event: object,
```

Comment:
Try avoiding types such as `object` or `any` since they aren't very strict. Here, we can type this as `React.ChangeEvent<{}>` or the whole constant as `AccordianProps['onChange']` (I prefer the latter)

Reply:
@Ry-DS I don't really get the `AccordianProps['onChange']` part, do you have an example that I can follow? for now I change it to `React.ChangeEvent<{}>`

Reply:
It would look something like:
```ts
const handleChange: (panel: string)=>AccordianProps['onChange'] = (panel)=>(event)=>{
// etc 
}
```